common/testutils/reporter.go

```diff
@@ -18,49 +18,49 @@ func NewReporter(t *testing.T, name string) *Reporter {
 	}
 }
 
-func (this *Reporter) Error(err error) {
-	namedFormat := this.formatName(err.Error())
-	this.T.Errorf(namedFormat)
+func (r *Reporter) Error(err error) {
```

Comment:
Removing more `this`s be like:
![](http://www.quickmeme.com/img/fb/fb4821a55878e400ce303c79cd0aff6a97bc6ac5e61181e7598338deff5bc8f5.jpg)